CONTRIBUTING.md

```diff
@@ -12,9 +12,49 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-#Adding a new section
+## Permissions & Contributions
 
-##Structure/Pattern
+There are two ways in which you can help update the content:
+
+* **One-off:** \
+If you are not a regular contributor to the project, but you would like to contribute some changes, the best way to do it is:
+
+  1. Fork the repo
+  2. Create a PR
+
+* **Periodic and regular contributions:** \
+If you plan to update the content semi-regularly or regularly, you can be added to the project's Contributors group. Please contact one of the project's administrators to be added to the group.
+
+## Git guidance
+
+Consistent with the practices suggested in this playbook, please follow the specifics regarding git as described in this section.
+
+### Branch naming convention
+
+In this repo, we use the following branch naming conventions:
```

Comment:
I think we should disallow branches other than master in this repo, and request that people use their own forks to make pull requests.

Reply:
Thoughts @TessFerrandez?

Reply:
I think that sounds good, especially as it is an OSS repo, but then I think it needs to be enforced as @rozele suggests to avoid the current branch situation

Reply:
I'll change the text to describe this approach. @TessFerrandez, @rozele - who has permissions to configure the repo accordingly?

Reply:
@MS-Santi @rozele @TessFerrandez I can change the policies on the repo if this is what we have decided. However, I am hesitating a little bit as although branches can clutter the repo (we can fix this by having an automated delete once a branch is merged) I think forks create much less visibility for what work is in progress as well as higher friction for contributions. Thoughts?

Reply:
@fnocera I was thinking the same today actually, as I was looking at some PRs. If you want to clone it locally to take it through the linter for example you have to create a separate folder, it was much more cumbersome than switching branches.

I would like to hear @rozele’s thoughts though. I was agreeing with him as it seemed like a good way to avoid all the leftover branches, but if we can add an automatic delete, that seems to solve that problem.  

If we keep the branches, I think it would be a good idea to have a common naming policy to understand what the branch is about.

Reply:
TBH I don't have a strong opinion on this, I think your comment regarding friction for contributors is very valid and perhaps that is the most important thing at this point

Reply:
@TessFerrandez @rozele @MS-Santi 
Eric, Tess and I had a discussion today and for the near term (next few months) we will not disable push permissions/force people to use forks. Once the number of issues/contributions to the playbook have leveled out we can move to a fork only model.  Therefore we do need branch naming conventions!

Reply:
Sounds good. I'll go back to the version with the naming conventions